src/rikin/assignment7/Shop.java

```diff
@@ -0,0 +1,131 @@
+package rikin.assignment7;
+/*Create a system (Shop) which maintains each item's quantity.
+a. Initial stocks of Maggie: 50 Maggie packets 
+b. Initial stocks of Dosa : 43 packets 
+c. Initial stocks of pouches : 39 oil packets 
+d. Initial stocks of panipuri : 43 packets  
+e. Initial stocks of masala�s : 73 packets.
+Expectation:
+1) Only one Customer came for shopping, he picked different items with different quantities, print "running out of stock" message if a particular item is sold out otherwise deduct requested quantity from total quantity. 
+2) Create a method which shows only those item which is "Out of stock".
+3) Create a method which shows only those item which is "Available in stock" & respective quantities.
+*/
+
+class Shop{
+	int maggieQty, dosaQty, oilPoucheQty, panipuriQty, masalaPacketQty;
+	 
+	void initialStocks(int maggie, int dosa, int oilPouche, int panipuri, int masalaPacket){
+		maggieQty = maggie;
+		dosaQty = dosa;
+		oilPoucheQty = oilPouche;
+		panipuriQty = panipuri;
+		masalaPacketQty = masalaPacket;
+			System.out.println("\n");
+			System.out.println("Items available at shop:");
+			System.out.println("Maggie stock quantity is " + maggieQty);
+			System.out.println("Dosa stock quantity is " + dosaQty);
+			System.out.println("Oil pouche stock quantity is " + oilPoucheQty);
+			System.out.println("panipuri stock quantity is " + panipuriQty);
+			System.out.println("Masala packet stock quantity is " + masalaPacketQty);			
+	}
+		
+	void requestedQty(int reqMaggie, boolean PMaggieSell, int reqDosa, boolean PDosaSell, int reqOilPouche, boolean POilPoucheSell, int reqPanipuri, boolean PPanipuriSell, int reqMasalaPacket, boolean PMasalaSell){
+		System.out.println("\n");
+		if(maggieQty ==0){
+			System.out.println("Sorry, Maggie is not available in stock.");
```

Comment:
Here, message should be different as if customer looking for 0 packs then why we should say that stock not available  right?